app/graphql/types/base/connection.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Types
+  module Base
+    class Connection < GraphQL::Types::Relay::BaseConnection
+      field :totalCount, Integer, null: false
+
+      def totalCount
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/e16fb7196db4de5a797840ca93bdb6f34b32271b/app/graphql/types/base/connection.rb#L8"></span>Has the name ‘totalcount’</td>
    </tr>
  </tbody>
</table>